react/features/base/util/uri.js

```diff
@@ -523,19 +523,25 @@ export function urlObjectToString(o: Object): ?string {
 
     // query/search
 
-    // Web's ExternalAPI jwt
-    const { jwt } = o;
+    // Web's ExternalAPI jwt and lang
+    const { jwt, lang } = o;
+
+    const search = new URLSearchParams(url.search);
 
     if (jwt) {
-        let { search } = url;
+        search.set('jwt', jwt);
+    }
 
-        if (search.indexOf('?jwt=') === -1 && search.indexOf('&jwt=') === -1) {
-            search.startsWith('?') || (search = `?${search}`);
-            search.length === 1 || (search += '&');
-            search += `jwt=${jwt}`;
+    const { defaultLanguage } = o.configOverwrite || {};
 
-            url.search = search;
-        }
+    if (lang || defaultLanguage) {
+        search.set('lang', lang || defaultLanguage);
+    }
+
+    const searchString = search.toString();
+
+    if (searchString) {
+        url.search = `?${searchString}`;
```

Comment:
Why not just assign `search` ?

Reply:
because search is of type `string` and not `URLSearchParams` apparently.

Reply:
<img width="198" alt="Screenshot 2022-03-08 at 13 39 08" src="https://user-images.githubusercontent.com/317464/157250180-dce7a1e9-9faa-493d-9240-5c3d2a1d2972.png">

That's not what I see.

Reply:
![image](https://user-images.githubusercontent.com/39557534/157257282-edd0d1c2-78a6-4606-b8d2-3a5ba8f52c7f.png)
![image](https://user-images.githubusercontent.com/39557534/157257511-849757a9-6e6e-4ca3-8960-baabe4a0a05a.png)
![image](https://user-images.githubusercontent.com/39557534/157257650-6b35c65a-2792-4fdd-ab41-df2e7af07737.png)


the `url` is a regular object in our case, it's not an URI.

Reply:
Ah crap, you're right.